setup.cfg

```
@@ -0,0 +1,22 @@
+[metadata]
+name = sphinxcontrib-towncrier
+url = https://github.com/sphinx-contrib/towncrier
+description = An RST directive for injecting a Towncrier-generated changelog draft containing fragments for the unreleased (next) project version
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Sviatoslav Sydorenko
+author_email = wk+github-open-source@sydorenko.org.ua
+license = BSD 3-Clause License
+license_file = LICENSE
```

According to https://wheel.readthedocs.io/en/stable/user_guide.html#including-license-files-in-the-generated-wheel-file, the singular option has been replaced with a plural one:
```suggestion
license_files =
    LICENSE
```